### ISSUE TITLE
Stopping extension should not stop other recording extensions - Allowing extension to run in parallel on more than one tab 

### ISSUE THREAD

**Issue by tsemachh** [webrecorder/archiveweb.page#127]
Hi
The researchers asked to allow them to open the extension on few tabs and to perform recording and stop the recording on different times.
Now , the problem here is with the detach API from the debugger , once we stop the recording in 1 extension we detach from the debugger causing other recordings to stop as well.
We added next snippet in our customized recorder and this resolved the issue:
```
  _doDetach() {
    const otherExtensionIsRunning =
      Object.values(self.recorders).filter((rec) => {
        return rec.tabId != this.tabId && rec.running;
      }).length > 0;
    return !otherExtensionIsRunning ? super._doDetach() : null;
  }
```
If it's relevant for other extension users you may correct it in the original extension.

Regards
Tsemach

**Comment by ikreymer** [comment]
Thanks, yes, this is a good idea, and quickly set this up in the linked PR. It required a bit more handling, including detaching event messages, to allow stopping/starting multiple times

**Comment by ikreymer** [comment]
Fixed in 0.9.6
